src/main/java/ca/on/oicr/gsi/migration/PinerySource.java

```
@@ -939,7 +941,18 @@ private boolean containsPairedEndLibraries(List<SequencerPoolPartition> lanes) {
           + "pool will contain %d dilutions. See Geospiza run %d, lane %d.",
               pool.getPoolableElements().size(), pos.getSamples().size(), dto.getId(), pos.getPosition())));
         }
-        String proj = ((MigrationLibrary) pool.getPoolableElements().iterator().next().getLibrary()).getProject().getShortName();
```

Was there a reason to change this? Project gets set on MigrationLibrary, so there is no need to get it from the Sample.

With the change, it will log an error message if a Library is migrated separately from its Sample, which is allowed to happen, and it will also skip the lane, but never cause migration to fail. If there is an error state, a ConversionException should be thrown instead of logging and ignoring